post-encryption.expect

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/expect -f
+set ovmf [lindex $argv 0]
+spawn sudo qemu-system-x86_64 -enable-kvm -nographic -m 1024 -cpu host -drive file=installer-target.img,if=virtio,aio=threads -net nic,model=virtio -net user,hostfwd=tcp::2233-:22 -smp 2 -bios $ovmf
```

Comment:
I'm not super in favor of doing this in the expect script but this isn't particularly problematic for now, we can come back to this another time.

Reply:
I don't see another way to test the root encrypted password during boot time, if you think in a way we could change in a future update